jupyter_scheduler/scheduler.py

```diff
@@ -243,7 +365,10 @@ def get_job(self, job_id: str) -> DescribeJob:
         with self.db_session() as session:
             job_record = session.query(Job).filter(Job.job_id == job_id).one()
 
-            return DescribeJob.from_orm(job_record)
+        model = DescribeJob.from_orm(job_record)
+        self.add_outputs(model=model)
+
+        return model
 
     def delete_job(self, job_id: str):
```

Comment:
Deleting a job should delete the corresponding files stored in the staging area.

Reply:
Not changing in this PR.